lib/Sema/TypeCheckDecl.cpp

```diff
@@ -3429,150 +3429,6 @@ static void validateTypealiasType(TypeChecker &tc, TypeAliasDecl *typeAlias) {
   typeAlias->setUnderlyingType(typeAlias->getUnderlyingTypeLoc().getType());
 }
 
-
-/// Bind the given function declaration, which declares an operator, to
```

Comment:
To make the diff smaller and easier to read, can you leave the ::evaluate() method in the same location as this function used to be? We can re-organize TypeCheckDecl.cpp later.